Tests/web3swiftTests/remoteTests/ENSTests.swift

```diff
@@ -79,10 +79,10 @@ class ENSTests: XCTestCase {
 
     func testTTL() async throws {
         let web3 = await Web3.InfuraMainnetWeb3(accessToken: Constants.infuraToken)
-        let ens = ENS(web3: web3)
+        let ens = try XCTUnwrap(ENS(web3: web3))
         let domain = "somename.eth"
-        let ttl = try await ens?.registry.getTTL(node: domain)
-        
+        let ttl = try await ens.registry.getTTL(node: domain)
+        XCTAssertGreaterThanOrEqual(ttl, 0)
```

Comment:
Your explanation makes sense here. Just one question so since it's `try` and the test function `throws`, this test would fail if there is an issue with getting the TTL for the domain, correct?

Reply:
Yes, `getTTL` will throw if arises any issue:
1. if nameHash couldn't be obtained
2. if transaction couldn't be created
3. if calling contract the result is nil
4. if result of getTTL doesn't contain "0" key
<img width="1398" alt="Captura de Pantalla 2023-01-05 a las 15 32 55" src="https://user-images.githubusercontent.com/912381/210805791-4dc9725d-771f-4c09-a79c-18d1619ee745.png">